### ISSUE TITLE
Plugins upsell nudges are confusing

### ISSUE THREAD

**Issue by mmtr** [Automattic/wp-calypso#65025]
### Quick summary

https://github.com/Automattic/wp-calypso/pull/64958 introduced a new upsell nudge for paid plugins, so we have now two nudges in the plugins screen: one for the paid plugins requiring the Starter plan, and another one for the free plugins requiring the Pro plan.

Paid plugins nudge | Free plugins nudge
--- | ---
<img width="500" alt="Screen Shot 2022-06-28 at 10 50 42" src="https://user-images.githubusercontent.com/1233880/176137209-d0278aa9-a8a9-4743-a234-f47cc2c5dcbb.png"> | <img width="500" alt="Screen Shot 2022-06-28 at 10 51 02" src="https://user-images.githubusercontent.com/1233880/176137284-31208643-f908-4f0c-a78d-a1106dd03847.png">

I found this layout a bit confusing, because it's not clear what plugins can be installed with the Pro plan. At first, it looks like the ones categorized as "Editor's pick" are included, while the ones categorized as "Top free plugins" don't need any plan at all.

Furthermore, the nudges are only visible in the "Discover" tab. As soon as I navigate away to a different tab, or if I enter a search term, the nudges are hidden (before https://github.com/Automattic/wp-calypso/pull/64958, there was always an upsell nudge at the top of the screen).

<img width="500" alt="Screen Shot 2022-06-28 at 10 42 32" src="https://user-images.githubusercontent.com/1233880/176138962-883a492c-4a7e-4d4d-8e73-49a30f7532a8.png">

### Steps to reproduce

- Go to https://wordpress.com/plugins?flags=marketplace-starter-plan
- Switch to a site with a Free plan

### What you expected to happen

The fact that "Starter plans can purchase paid plugins, but must upgrade to Pro to use free plugins" should be clear and easy to understand.

### What actually happened

The nudges can confuse users about what plan is required to use a plugin.

**Comment by mreishus** [comment]
I noticed the same thing. See my comment here: https://github.com/Automattic/wp-calypso/pull/64958#issuecomment-1167801319

I also think `"Starter plans can purchase paid plugins, but must upgrade to Pro to use free plugins"` may be a hard concept to get across in general, and possibly needs tweaked wording. Paying more to access something free just doesn't sound right. :)

**Comment by DustyReagan** [comment]
I think calling them "paid plugins" and "free plugins" compounds the confusion. My two cents is to rename "paid plugins" to "premium plugins" and "free plugins" to "community plugins". Or if not those names, let's at least get the words "paid" and "free" out of the names somehow.

Then do something like this on the Browse page:
<img src="https://user-images.githubusercontent.com/140841/176242723-4c73b08c-5433-49be-ad62-a0a0773ec03d.png" width="400" />

**Comment by vinimotaa** [comment]
Hi folks,

thanks for bringing this for discussion @mmtr!

@SaxonF and I are accountable for design decisions on Marketplace. Yesterday during our 1/1 we exchanged some ideas around this issue, specially all the complexity around different plans permissions when it comes to installing paid/free plugins. We believe that **the best to do now is to remove/hide any UI/alert message regarding plan upgrade from the _Marketplace landing page_**, and here are some of our rationals;

 - Beyond confusing, we believe the landing page isn't the best place to display those messages as we'd like the users to keep exploring the plugins without any friction
 - The UI seems like an alert/blocker message that can bounce users off
 - We want to create plan upgrade awareness on a **plugin level** (the moment they make a decision "I need this") so then we can understand (learn) what is the user need (maybe use that to upsell the "plan upgrade" from marketing the plugin's benefits to this specific user?)
 - We want to increase the plan upgrade conversion from the Marketplace by providing a better contextualization (What's going on, why do I need another plan, what do I get and how much do I need to pay)
 
 With that, this UI request became an entire UX/flow design exercise that I'm working on, considering all the different use cases, and this will become a P2 post with an initial POC for us to extend this discussion with the broader team and move from there.
 
 We're also [looking for/requesting data from this existing UI](https://a8c.slack.com/archives/C02V49GM0PN/p1656513253396239?thread_ts=1656428470.302489&cid=C02V49GM0PN) so we can make more informed decisions.
 
 cc @ianstewart @obenland

**Comment by mmtr** [comment]
I just deployed https://github.com/Automattic/wp-calypso/pull/65123 which removes the upgrade banners from the Plugins browser page. 

@vinimotaa Do you want me to keep this issue open for tracking the remaining UX/flow design work?

**Comment by vinimotaa** [comment]
yes @mmtr, please!

I'll communicate all this on a P2 once that is finished, and we take it from there!

**Comment by vinimotaa** [comment]
Hi folks, here is an update on this issue:

pdh6GB-1uD-p2

**Comment by cpapazoglou** [comment]
Closing this as plugins will be only available to pro, biz, ecom plans. pdtkmj-io-p2#comment-532
